test/inspector/test-inspector-port-cluster.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const assert = require('assert');
+const cluster = require('cluster');
+const common = require('../common');
+const debuggerPort = common.PORT;
+const childProcess = require('child_process');
+
+common.skipIfInspectorDisabled();
+
+let offset = 0;
+
+/*
+ * This test suite checks that inspector port in cluster is incremented
+ * for different execArgv combinations
+ */
+
+function testRunnerMain() {
+  spawnMaster({
+    execArgv: ['--inspect'],
+    workers: [{expectedPort: 9230}]
+  });
+
+  let port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: [`--inspect=${port}`],
+    workers: [
+      {expectedPort: port + 1},
+      {expectedPort: port + 2},
+      {expectedPort: port + 3}
+    ]
+  });
+
+  port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: ['--inspect', `--inspect-port=${port}`],
+    workers: [{expectedPort: port + 1}]
+  });
+
+  port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: ['--inspect', `--debug-port=${port}`],
+    workers: [{expectedPort: port + 1}]
+  });
+
+  port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: [`--inspect=0.0.0.0:${port}`],
+    workers: [{expectedPort: port + 1, expectedHost: '0.0.0.0'}]
+  });
+
+  port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: [`--inspect=127.0.0.1:${port}`],
+    workers: [{expectedPort: port + 1, expectedHost: '127.0.0.1'}]
+  });
+
+  if (common.hasIPv6) {
+    port = debuggerPort + offset++ * 10;
+
+    spawnMaster({
+      execArgv: [`--inspect=[::]:${port}`],
```

Comment:
Ohh yeah, the bug was just that, if the hostname is not an IPv4 address resolution falls back to 127.0.0.1. https://github.com/nodejs/node/issues/13477
So this might change after #13478.